tests/errors/well-formed/invalid-attributes.expect

```diff
@@ -0,0 +1,6 @@
+---CODE---
+101
+---STDERR---
+thread 'main' panicked at crates/ast/src/parser.rs:828:96:
+Found unknown attribute flag "invalid_attribute"
```

Comment:
Can we make the the compiler generate a nice error message like the Calyx frontend does: https://github.com/calyxir/calyx/blob/main/calyx-frontend/src/parser.rs#L282